### PR TITLE
Pablo.quintana/exercise02

### PR DESCRIPTION
**Pull request by wablopilson**
I tested the routes using a Postman collection and newman in iTerm. All the documentation is included in README file.

### CODE REVIEW

File: message/utils/checkIfEmpty.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const checkIfEmpty = (str) => {
+
+    if (str === '') return true
+
+    const strArr = str.split('')
+
+    for (let i = 0; i < strArr.length; i++) {
```
Comment (joskar) [inline message/utils/checkIfEmpty.js:7]: @wablopilson  Can you please expalin the usecase that you are trying solve here?

Comment (wablopilson) [reply message/utils/checkIfEmpty.js:7]: I created that util to check if any string you send to that function coming from a request (in our case, message or destination) is empty or is just filled with spaces.

File: message/routes/messages.routes.js [hunk 2]
```diff
@@ -11,10 +13,27 @@ router.post("/", (req, res, next) => {
 
     const { destination, message } = req.body
 
-    messageService
-        .createMessage({ destination, body: message })
-        .then(response => res.status(200).json(response.data))
-        .catch(err => res.status(500).json(err))
+
+    if (!req.body.destination) {
+        res.status(400).json({ message: 'Destination field is required' })
```
[the remark this answers is not in the document]
Comment (wablopilson) [reply message/routes/messages.routes.js:18]: Yeah! I'll check that site out so I can update the status to make them more specific, thanks!

[the remark this answers is not in the document]
Comment (wablopilson) [reply message/routes/messages.routes.js]: updated!

**Pull request merged**